.azure-pipelines/testscripts_analyse/analyse_testscripts.py

```diff
@@ -40,7 +40,7 @@
 
 
 def topo_name_to_type(topo_name):
-    pattern = re.compile(r'^(wan|t0|t1|ptf|fullmesh|dualtor|t2|tgen|mgmttor|m0|mc0|mx|dpu|any)')
+    pattern = re.compile(r'^(wan|t0|t1|ptf|fullmesh|dualtor|t2|tgen|mgmttor|m0|mc0|mx|dpu|any|multidut-tgen|snappi)')
```

Comment:
The pattern here indeed is topo_type before normalization.
Why not just use `tgen` here?

Reply:
In topo map, multidut-tgen is calculated as tgen, here is the place to match topo, we need to match multidut-tgen here, and map to tgen in later steps

Reply:
Hi, @wangxin, here is pattern to math the `pytest.mark.topology` in script. Although topo name multi-tgen belongs to topo type tgen, but some scripts use this name rather than the type. To avoid the warning `WARNING:root:Unsupported testbed type - multidut-tgen`, I think it's better to add this name into this pattern.

Reply:
Move `multidut-tgen` to just after `tgen`?

Reply:
Fixed